dashboard/config/marketing/announcements.json

```diff
@@ -1,6 +1,6 @@
 {
   "pages": {
-    "/home": "music-lab-launch",
+    "/home": "lms-launch",
```

Comment:
Changing this now will remove the Music Lab banner, and no banner will show up until the `show-updated-lms-content` DCDO flag is flipped. I assume we want to keep the Music Lab banner up until this LMS banner goes live, but @dmcavoy can you confirm?

Reply:
We've had the music lab banner up for 2 weeks. I think its alright for it to come down when this goes in

Reply:
Ok cool, ty! @TurnerRiley this is fine as-is 👍